vnext/ReactWindowsCore/Modules/UIManagerModule.cpp

```diff
@@ -492,6 +502,11 @@ std::vector<facebook::xplat::module::CxxModule::Method> UIManagerModule::getMeth
       Method(
           "measureInWindow",
           [manager](dynamic args, Callback cb) { manager->measureInWindow(jsArgAsInt(args, 0), cb); }),
+      Method(
+          "measureLayout",
+          [manager](dynamic args, Callback cbError) {
```

Comment:
>[manager](dynamic args, Callback cbError) { [](start = 10, length = 43)

It does not look right: measure Layout method supposed to get two callbacks, while this code exposes it to JavaScript as a method with one callback treating it as an error callback for success and failure cases.

Reply:
Good catch. I was testing something and must've forgotten to change this back...

Now when I try to add the 3rd callback parameter, this method won't get called (added break point at L508 and it was never hit), there was no error on js side either. Any idea what the issue might be?

Reply:
Problem was the callback functions were passed in as promises and we need to add AsyncTag mark to the function registration method. Thanks @jonthysell for taking a look!